recipes/diligent-fx/all/conandata.yml

```diff
@@ -0,0 +1,14 @@
+sources:
+  "cci.20211112":
```

Comment:
does really need this old version? otherwise, I recommend only supporting the latest.

Reply:
It was referenced in Diligent-Engine together with diligent-core/2.5.1@
I would prefer to keep this version and add the most actual one in parallel :)

Reply:
So, I will add the latest version as soon as this one is merged :)